express.js

```diff
@@ -138,6 +145,10 @@ var createApiWidgetInstanceData = id => {
 var createApiWidgetData = (id) => {
 	let widget = yaml.parse(getInstall().toString());
 
+	//provide default values where necessary
+	if ( ! widget.meta_data.features) widget.meta_data.features = [];
+	if ( ! widget.meta_data.supported_data) widget.meta_data.features = [];
```

Comment:
New feature: MWDK will now handle empty features and supported_data in demo.json to more closely mimic the server.